src/hooks/chat.test.ts

```diff
@@ -61,4 +73,58 @@ describe('Chat Hooks', () => {
       expect(queryClient.getQueryData(key)).toBeFalsy();
     });
   });
+
+  describe('useItemChat with arguments', () => {
+    const itemId = ITEMS[0].id;
+    const route = `/${buildGetItemChatRoute(itemId)}`;
+    const key = buildItemChatKey(itemId);
+
+    it(`getUpdates = true`, async () => {
+      const hook = () => hooks.useItemChat(itemId, { getUpdates: true });
+      const response = {
+        id: itemId,
+        messages: buildChatMessages(itemId),
+      };
+      const endpoints = [
+        {
+          route,
+          response,
+        },
+      ];
+      const { data } = await mockHook({
+        endpoints,
+        hook,
+        wrapper,
+      });
+
+      expect((data as List<ChatMessage>).toJS()).toEqual(response);
+
```

Comment:
Is there a way to test if the websockets did something? otherwise this test shows only that it doesn't throw.